simulation/traffic_simulator/include/traffic_simulator/api/api.hpp

```diff
@@ -126,7 +117,10 @@ class API
         simulation_api_schema::SpawnVehicleEntityResponse res;
         simulation_interface::toProto(parameters, *req.mutable_parameters());
         req.mutable_parameters()->set_name(name);
+        req.set_asset_key(model3d);
+        simulation_interface::toProto(toMapPose(pose), *req.mutable_pose());
         req.set_is_ego(behavior == VehicleBehavior::autoware());
+        req.set_initial_speed(0.0);  // TODO: Should be filled from function API
```

Comment:
Please doxygen @todo comment.
```suggestion
        /// @todo Should be filled from function API
        req.set_initial_speed(0.0);
```

Reply:
done in cf617e005b9ccc13c8ff76b236c97972c4f9838c